<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>691</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>10</width>
    <height>10</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MuseumNet GUI</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../.designer/backup/cam.ico</normaloff>
    <normalon>../../.designer/backup/cam.ico</normalon>../../.designer/backup/cam.ico</iconset>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>761</width>
         <height>689</height>
        </rect>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QFrame" name="topBar">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>6</number>
           </property>
           <property name="rightMargin">
            <number>6</number>
           </property>
           <property name="bottomMargin">
            <number>5</number>
           </property>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="minimizeButton">
             <property name="maximumSize">
              <size>
               <width>25</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="iconSize">
              <size>
               <width>8</width>
               <height>8</height>
              </size>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="maximizeButton">
             <property name="maximumSize">
              <size>
               <width>25</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="iconSize">
              <size>
               <width>8</width>
               <height>8</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="closeButton">
             <property name="maximumSize">
              <size>
               <width>25</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="iconSize">
              <size>
               <width>8</width>
               <height>8</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QFrame" name="leftFrame">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <property name="lineWidth">
              <number>0</number>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>20</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>APP</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="menuFrame" native="true">
                <layout class="QVBoxLayout" name="verticalLayout_3">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QPushButton" name="databaseButton">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>35</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <pointsize>11</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>Database</string>
                   </property>
                   <property name="iconSize">
                    <size>
                     <width>30</width>
                     <height>15</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="analysisButton">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>35</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <pointsize>11</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>Analysis</string>
                   </property>
                   <property name="iconSize">
                    <size>
                     <width>30</width>
                     <height>18</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="verticalSpacer">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>40</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="mainFrame" native="true">
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QStackedWidget" name="stackedWidget">
                <widget class="QWidget" name="page">
                 <layout class="QVBoxLayout" name="verticalLayout_5">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QLabel" name="p1header">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>40</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>40</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <pointsize>11</pointsize>
                     </font>
                    </property>
                    <property name="text">
                     <string>   Database management</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QWidget" name="p1content" native="true">
                    <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0" columnstretch="0,0">
                     <property name="leftMargin">
                      <number>25</number>
                     </property>
                     <property name="topMargin">
                      <number>20</number>
                     </property>
                     <property name="rightMargin">
                      <number>25</number>
                     </property>
                     <property name="bottomMargin">
                      <number>10</number>
                     </property>
                     <property name="horizontalSpacing">
                      <number>40</number>
                     </property>
                     <property name="verticalSpacing">
                      <number>10</number>
                     </property>
                     <item row="0" column="1">
                      <widget class="QLabel" name="label_2">
                       <property name="text">
                        <string>Database:</string>
                       </property>
                      </widget>
                     </item>
                     <item row="1" column="1">
                      <widget class="QLabel" name="label_3">
                       <property name="text">
                        <string>Identities:</string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="0">
                      <widget class="QPushButton" name="loadButton">
                       <property name="minimumSize">
                        <size>
                         <width>0</width>
                         <height>25</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>125</width>
                         <height>16777215</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>  Load Database</string>
                       </property>
                      </widget>
                     </item>
                     <item row="2" column="0" colspan="2">
                      <widget class="QTabWidget" name="dbTab">
                       <property name="minimumSize">
                        <size>
                         <width>0</width>
                         <height>500</height>
                        </size>
                       </property>
                       <property name="currentIndex">
                        <number>1</number>
                       </property>
                       <property name="iconSize">
                        <size>
                         <width>15</width>
                         <height>15</height>
                        </size>
                       </property>
                       <widget class="QWidget" name="tab_2">
                        <attribute name="title">
                         <string>Database Browser</string>
                        </attribute>
                       </widget>
                       <widget class="QWidget" name="addTab">
                        <property name="enabled">
                         <bool>true</bool>
                        </property>
                        <attribute name="title">
                         <string>Add Person</string>
                        </attribute>
                        <layout class="QGridLayout" name="gridLayout_2">
                         <item row="0" column="1">
                          <widget class="QPushButton" name="addImageButton">
                           <property name="minimumSize">
                            <size>
                             <width>110</width>
                             <height>25</height>
                            </size>
                           </property>
                           <property name="maximumSize">
                            <size>
                             <width>110</width>
                             <height>16777215</height>
                            </size>
                           </property>
                           <property name="text">
                            <string>From image</string>
                           </property>
                          </widget>
                         </item>
                         <item row="0" column="0">
                          <widget class="QPushButton" name="addCameraButton">
                           <property name="minimumSize">
                            <size>
                             <width>110</width>
                             <height>25</height>
                            </size>
                           </property>
                           <property name="maximumSize">
                            <size>
                             <width>110</width>
                             <height>16777215</height>
                            </size>
                           </property>
                           <property name="text">
                            <string>From camera</string>
                           </property>
                          </widget>
                         </item>
                         <item row="0" column="2">
                          <spacer name="horizontalSpacer_2">
                           <property name="orientation">
                            <enum>Qt::Horizontal</enum>
                           </property>
                           <property name="sizeHint" stdset="0">
                            <size>
                             <width>40</width>
                             <height>20</height>
                            </size>
                           </property>
                          </spacer>
                         </item>
                         <item row="1" column="0" colspan="3">
                          <widget class="QStackedWidget" name="stackedWidget_2">
                           <widget class="QWidget" name="page_3">
                            <layout class="QVBoxLayout" name="verticalLayout_6">
                             <item>
                              <widget class="QLabel" name="imageDisplay">
                               <property name="text">
                                <string/>
                               </property>
                              </widget>
                             </item>
                             <item>
                              <widget class="QComboBox" name="comboBox">
                               <property name="enabled">
                                <bool>true</bool>
                               </property>
                               <property name="maximumSize">
                                <size>
                                 <width>200</width>
                                 <height>16777215</height>
                                </size>
                               </property>
                              </widget>
                             </item>
                             <item>
                              <widget class="QPushButton" name="captureButton">
                               <property name="minimumSize">
                                <size>
                                 <width>0</width>
                                 <height>20</height>
                                </size>
                               </property>
                               <property name="maximumSize">
                                <size>
                                 <width>200</width>
                                 <height>16777215</height>
                                </size>
                               </property>
                               <property name="text">
                                <string>Capture</string>
                               </property>
                              </widget>
                             </item>
                            </layout>
                           </widget>
                           <widget class="QWidget" name="page_4"/>
                          </widget>
                         </item>
                        </layout>
                       </widget>
                      </widget>
                     </item>
                     <item row="1" column="0">
                      <widget class="QPushButton" name="saveButton">
                       <property name="minimumSize">
                        <size>
                         <width>0</width>
                         <height>25</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>125</width>
                         <height>16777215</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>  Save Database</string>
                       </property>
                      </widget>
                     </item>
                     <item row="3" column="0">
                      <spacer name="verticalSpacer_2">
                       <property name="orientation">
                        <enum>Qt::Vertical</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>20</width>
                         <height>40</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </widget>
                <widget class="QWidget" name="page_2"/>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
